src/rtc/h5_prep.py

```diff
@@ -300,20 +309,195 @@ def get_metadata_dict(product_id: str,
-        #     ["https://ceos.org/ard/files/PFS/NRB/v5.5/CARD4L-PFS_NRB_v5.5.pdf",
-        #      'CEOS document identifier'],
+             ['software_version', str(SOFTWARE_VERSION), 'Software version'],
+        #'identification/CEOSDocumentIdentifier':
+        #    ["https://ceos.org/ard/files/PFS/NRB/v5.5/CARD4L-PFS_NRB_v5.5.pdf",
+        #     'Product version'],
+        'RTC/metadata/sourceDataInformation/numberOfAcquisitions': # placeholder
+            ['number_of_acquisitions',
+             0,
+             'Number of source data acquisitions'],
+        'RTC/metadata/sourceDataInformation/dataAccess':
+            ['data_access',
+             'https://search.asf.alaska.edu/',
+             'Where the source data can be retrieved'],
+        'RTC/metadata/sourceDataInformation/radarBand': # 1.6.4
+            ['radar_band', 'C', 'Radar band'],
+        'RTC/metadata/sourceDataInformation/processingFacility': # 1.6.6
+            ['source_processing_facility',
+             'Sentinel-1 Instrument Processing Facility (IPF)',
+             'Source data processing facility'],
+        'RTC/metadata/sourceDataInformation/processingDateTime': # 1.6.6
+            ['source_processing_date_time',
+             burst_in.burst_misc_metadata.processing_date.strftime("%Y-%m-%dT%H:%M:%S.%fZ"),
+             'Processing date and time of the source data'],
+        'RTC/metadata/sourceDataInformation/processingSoftwareVersion': # 1.6.6
+            ['source_processing_sw_version',
+             str(burst_in.ipf_version),
+             'IPF version of the source data'],
+        
+        'RTC/metadata/sourceDataInformation/azimuthLooks': # 1.6.6
+            ['source_azimuth_looks',
+             burst_in.burst_misc_metadata.azimuth_looks,
+             'Azimuth number of looks'],
+        'RTC/metadata/sourceDataInformation/slantRangeLooks': # 1.6.6
+            ['source_slant_range_looks',
+             burst_in.burst_misc_metadata.slant_range_looks,
+             'Slant range number of looks'],
+
+        'RTC/metadata/sourceDataInformation/productLevel': # 1.6.6
+            ['source_product_level',
+             'L1',
+             'Product level of the source data'],
+        'RTC/metadata/sourceDataInformation/geometry': # 1.6.7
+            ['source_geometry',
+             'slant range',
+             'Geometry of the source data'],
+        'RTC/metadata/sourceDataInformation/azimuthSpacing': # 1.6.7
+            ['source_azimuth_spacing',
+             burst_in.azimuth_time_interval,
+             'Azimuth spacing of the source data in seconds'], 
+        'RTC/metadata/sourceDataInformation/slantRangeSpacing': # 1.6.7
+            ['source_slant_range_spacing',
+             burst_in.range_pixel_spacing,
+             'Slant range spacing of the source data in meters'], 
+
+        'RTC/metadata/sourceDataInformation/azimuthResolution': # 1.6.7
+            ['source_azimuth_resolution',
+             burst_in.azimuth_time_interval,
```

Comment:
Please note that `*resolution` have the same values as `*spacing`

Reply:
If we don't have resolution values, let's comment these lines for now (we should populate them later with correct values).